test/socket.test.ts

```diff
@@ -93,14 +92,16 @@ describe("SocketProxyProvider", () => {
 
   it("should work without a proxy", async () => {
     server.write("server->client")
-    assert.equal(await getData(fromServerToClient), "server->client")
+    const dataFromServerToClient = await (await getData(fromServerToClient)).toString()
```

Comment:
Ah! Probably not. Good catch! I'll take a closer look

Reply:
Updated the types and removed the unnecessary `await` - good catch! 

Also, seems I have to convert `Buffer` to `string` or I get back characters? 🤷‍♂️ I'll leave that part as is..

<img width="944" alt="image" src="https://user-images.githubusercontent.com/3806031/105777306-5d620400-5f27-11eb-8b71-5256f5b0391d.png">

Reply:
Yup! You'll still need the `toString`. I guess the `assert` library automatically does that conversion for some reason. Or maybe it's because it wasn't a strict equals.